src/overlays/fbdemos/ovl_fbdemo_triforce/z_fbdemo_triforce.c

```diff
@@ -31,8 +31,11 @@ void TransitionTriforce_Start(void* thisx) {
         case STATE_SPIRAL_IN_FAST:
             this->transPos = 1.0f;
             return;
+
+        default:
+            this->transPos = 0.03f;
+            return;
```

Comment:
Can these returns be made into `break;`'s now?

Reply:
No they cannot :(